packages/lit-dev-content/site/docs/components/decorators.md

```diff
@@ -10,7 +10,7 @@ versionLinks:
 
 Decorators are special functions that can modify the behavior of classes, class methods, and class fields. Lit uses decorators to provide declarative APIs for things like registering elements, reactive properties, and queries.
 
-Decorators are a [stage 2 proposal](https://github.com/tc39/proposal-decorators) for addition to the ECMAScript standard, which means they're neither finalized nor implemented in browsers yet. Compilers like [Babel](https://babeljs.io/) and [TypeScript](https://www.typescriptlang.org/) provide support for proposed features like decorators by compiling them into standard JavaScript a browser can run.
+Decorators are a [stage 3 proposal](https://github.com/tc39/proposal-decorators) for addition to the ECMAScript standard, which means they're neither finalized nor implemented in browsers yet. Compilers like [Babel](https://babeljs.io/) and [TypeScript](https://www.typescriptlang.org/) provide support for proposed features like decorators by compiling them into standard JavaScript a browser can run.
```

Comment:
I think a stage 3 proposal is considered finalized, save for any last minute issues that come up as browsers and compilers implement it.

Reply:
https://tc39.es/process-document/
At stage 3, the spec is considered "complete" but at stage 4 is when it's considered "final" so I think our wording here of "neither finalized nor implemented" is still valid.

Reply:
TC39's definition of "complete" and "final" are a bit different in practice than common understanding though. Stage 3 means that, barring drastic implementor feedback, there will not be any changes. It's basically final in 99% of cases, which is why this is the stage where implementors implement and ship.

Reply:
Updated text, added an aside to describe the process a little bit, because having all of this packed into one paragraph was getting clumsy.